### DISCUSSION TITLE
Monto establecido de egresos

### DISCUSSION THREAD

**Discussion by julchat** [dds-jv/foro#154] [TPA - 2020] [answered]
Hola,
> Bloquear la posibilidad de aceptar nuevos egresos a una determinada entidad, porque la misma superó un monto previamente establecido.

Se refiere a una cantidad máxima de egresos, o un monto de dinero máximo que tiene una entidad para gastar, y con el costo de los demás egresos, esta suma no debería exceder lo establecido?

Gracias

**Comment by SofiCortes** [comment, marked as answer]
Hola @julchat!

Sería un monto de dinero máximo para gastar. De todas formas, si lo interpretaron como cantidad de egresos, también es válido.

**Comment by julchat** [comment]
Gracias!
